lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Main.TcPOU

```diff
@@ -0,0 +1,109 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+  <POU Name="Main" Id="{3ee7fb27-9ab5-458e-ba31-7ac39aacb9ee}" SpecialFunc="None">
+    <Declaration><![CDATA[PROGRAM Main
+VAR
+	// Axes for each system
+	// M1L0
+	M1 : DUT_MotionStage:=(nCommand:=3); // Yup
+	fbMotionStage_m1 : FB_MotionStage;
+
+	M2 : DUT_MotionStage:=(nCommand:=3); // Ydwn
+	fbMotionStage_m2 : FB_MotionStage;
+
+	M3 : DUT_MotionStage:=(nCommand:=3); // Xup
+	fbMotionStage_m3 : FB_MotionStage;
+
+	M4 : DUT_MotionStage:=(nCommand:=3); // Xdwn
+	fbMotionStage_m4 : FB_MotionStage;
+
+	M5 : DUT_MotionStage:=(nCommand:=3); // PitchCoarse
+	fbMotionStage_m5 : FB_MotionStage;
+
+	M6 : DUT_MotionStage:=(nCommand:=3); // Bender
+	fbMotionStage_m6 : FB_MotionStage;
+	
+	fbRunHOMS_M1L0 : FB_RunHOMS;
+	
+	bExecuteCoupleYM1L0 : BOOL;
+	bExecuteDecoupleYM1L0 : BOOL;
+	bExecuteCoupleXM1L0 : BOOL;
+	bExecuteDecoupleXM1L0 : BOOL;
+	
+	bGantryAlreadyCoupledYM1L0 : BOOL;
+	bGantryAlreadyCoupledXM1L0 : BOOL;
+	
+	nCurrGantryYM1L0 : LINT; // encoder counts = nm
+	nCurrGantryXM1L0 : LINT; // encoder counts = nm
+	nGantryTolYM1L0 : LINT := 50000; // encoder counts = nm
+	nGantryTolXM1L0 : LINT := 50000; // encoder counts = nm
+	
+	// M2L0
+	M7 : DUT_MotionStage:=(nCommand:=3); // Yup
+	fbMotionStage_m7 : FB_MotionStage;
+
+	M8 : DUT_MotionStage:=(nCommand:=3); // Ydwn
+	fbMotionStage_m8 : FB_MotionStage;
+
+	M9 : DUT_MotionStage:=(nCommand:=3); // Xup
+	fbMotionStage_m9 : FB_MotionStage;
+
+	M10 : DUT_MotionStage:=(nCommand:=3); // Xdwn
+	fbMotionStage_m10 : FB_MotionStage;
+
+	M11 : DUT_MotionStage:=(nCommand:=3); // PitchCoarse
+	fbMotionStage_m11 : FB_MotionStage;
+
+	M12 : DUT_MotionStage:=(nCommand:=3); // Bender
+	fbMotionStage_m12 : FB_MotionStage;
+
+	fbRunHOMS_M2L0 : FB_RunHOMS;
+
+	bExecuteCoupleYM2L0 : BOOL;
+	bExecuteDecoupleYM2L0 : BOOL;
+	bExecuteCoupleXM2L0 : BOOL;
+	bExecuteDecoupleXM2L0 : BOOL;
+	
+	bGantryAlreadyCoupledYM2L0 : BOOL;
+	bGantryAlreadyCoupledXM2L0 : BOOL;
+	
+	nCurrGantryYM2L0 : LINT; // encoder counts = nm
+	nCurrGantryXM2L0 : LINT; // encoder counts = nm
+	nGantryTolYM2L0 : LINT := 50000; // encoder counts = nm
+	nGantryTolXM2L0 : LINT := 50000; // encoder counts = nm
+END_VAR]]></Declaration>
+    <Implementation>
+      <ST><![CDATA[// M1L0
+fbRunHOMS_M1L0(stYup:=M1, stYdwn:=M2, stXup:=M3, stXdwn:=M4, stPitch:=M5, stBender:=M6,
```

Comment:
You might consider putting only one argument per line, as these blocks of 20 arguments to the `fbRunHOMS_M1L0` call can be a bit tough to read through.

Reply:
Resolved in commit 69e2191